src/other/Table.js

```diff
@@ -23,11 +23,12 @@
 
     Table.prototype.publish("renderHtmlDataCells", false, "boolean", "enable or disable HTML within cells",null,{tags:["Private"]});
     Table.prototype.publish("pagination", false, "boolean", "Enable or disable pagination",null,{tags:["Private"]});
+    Table.prototype.publish("paginationLimit", null, "number", "Maximum number of rows allowed before pagination defaults to on",null,{tags:["Private"]});
```

Comment:
Does the Table have the ability to force a specific number of rows per page? (I remember we decided to allow the paginator to strictly set the number of rows per page in order to allow room for the page controls at the bottom) 

If we can't set the rows-per-page specifically then I think paginationLimit is a better approach because it allows the table to grow reasonably large without forcing pagination.